app/code/community/ShipStream/Sync/Model/Cron.php

```diff
@@ -28,10 +28,21 @@ public function fullInventorySync($sleep = TRUE)
                     $db->beginTransaction();
                     try {
                         $target = $this->_getTargetInventory(array_keys($source));
+                        //Get qty of order items that are in processing state and not submitted to shipstream
+                        $processingQty = $this->_getProcessingOrderItemsQty(array_keys($source));
                         foreach ($source as $sku => $qty) {
-                            if ( ! isset($target[$sku])) continue;
-                            if (floatval($qty) === floatval($target[$sku]['qty'])) continue;
-                            Mage::log("SKU: $sku remote qty is $qty and local is {$target[$sku]['qty']}", Zend_Log::DEBUG, self::LOG_FILE);
+                            if ( ! isset($target[$sku])) {
+                                continue;
+                            }
+                            $syncQty = $qty =  floor(floatval($qty));
```

Comment:
Accidental assignment of `$qty`?

Reply:
`$qty` is storing floor of qty in case as suggested on Clickup ticket `For inventory sync use floor operation on the quantities as there are cases where ShipStream may have a decimal quantity but the Magento product does not expect there to be decimals - such as "cases" of products that have been split at the warehouse but can only be ordered in whole quantities by the customer`